FLEDGE.md

```diff
@@ -130,10 +131,10 @@ const myGroup = {
                    {renderUrl: gymShoes, sizeGroup; 'group2', ...},
                    {renderUrl: gymTrainers1, sizeGroup: 'size4', ...},
                    {renderUrl: gymTrainers2, sizeGroup: 'size4', ...}],
-  'adSizes': {'size1': {width: width1, height: height1},
-              'size2': {width: width2, height: height2},
-              'size3': {width: width3, height: height3},
-              'size4': {width: width4, height: height4}},
+  'adSizes': {'size1': {width: '100', height: '100'},
+              'size2': {width: '100px', height: '200px'},
+              'size3': {width: '75sw', height: '25sh'},
+              'size4': {width: '100', height: '25sh'}},
```

Comment:
I think all these different units are a little more distracting than helpful.  Maybe just list two ad sizes here, one matching this requestedSize and one not?  The available units can go down in the text where you talk about normalization.

Reply:
I'm not going to trim or rework the sizeGroups section in this CL, as I have no idea what a size group is, and that's really beyond the scope of this PR, so I am not going to trim down the adSizes block.

This PR is also not introducing the ad sizes units, and I'm not really very familiar with them, anyways, so I'm reluctant to dig into them here.  I've just removed all the units.  I'll leave mucking with them for another PR.